<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QMainWindow" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decryption</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>950</width>
      <height>670</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
background-color: rgb(248, 248, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14px;
background-color: rgb(51, 156, 255);
color : #ffffff
</string>
      </property>
      <property name="text">
       <string> Image Options </string>
      </property>
     </widget>
     <widget class="QPushButton" name="select_IMG_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 10pt &quot;Calibri&quot;;
	border : 1px solid rgb(51, 156, 255);
	
	background-color: rgb(255, 255, 255);
	border-radius : 10px;
}
QPushButton::hover {
	font: 12pt &quot;Calibri&quot;;
	background-color: rgb(51, 156, 255);
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Select Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/photos/Photos/png.png);
background-color: rgb(248, 248, 255);
border : 1px solid rgb(248, 248, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>select_IMG_btn</zorder>
     <zorder>label_4</zorder>
     <zorder>groupBox_11</zorder>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
background-color: rgb(248, 248, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14px;
background-color: rgb(51, 156, 255);
color : #ffffff
</string>
      </property>
      <property name="text">
       <string>    Encryption</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);
border :0px solid rgb(248, 248, 255);
</string>
      </property>
      <property name="text">
       <string>Enter the Key :</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textbox_key">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit {
	background-color: rgb(255, 255, 255);
	font: 9pt &quot;Calibri&quot;;
}

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/photos/Photos/key.png);
border : 1px solid rgb(248, 248, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>460</y>
       <width>221</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
background-color: rgb(248, 248, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14px;
background-color: rgb(51, 156, 255);
color : #ffffff
</string>
      </property>
      <property name="text">
       <string>       Options </string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 10pt &quot;Calibri&quot;;
	border : 1px solid rgb(51, 156, 255);
	
	background-color: rgb(255, 255, 255);
	border-radius : 10px;
}
QPushButton::hover {
	font: 12pt &quot;Calibri&quot;;
	background-color: rgb(51, 156, 255);
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/photos/Photos/left-arrow.png);
border : 1px solid rgb(248, 248, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>621</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
background-color: rgb(248, 248, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14px;
background-color: rgb(51, 156, 255);
color : #ffffff
</string>
      </property>
      <property name="text">
       <string>   Informations and Progress</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>601</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border : 1px solid rgb(51, 156, 255);
border-radius : 10px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
background-color: rgb(51, 156, 255);
color : #ffffff
</string>
       </property>
       <property name="text">
        <string>          Info</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 09pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);
border :0px solid rgb(248, 248, 255);
</string>
       </property>
       <property name="text">
        <string>Name                :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 09pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);
border :0px solid rgb(248, 248, 255);
</string>
       </property>
       <property name="text">
        <string>Size                    :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 09pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);
border :0px solid rgb(248, 248, 255);
</string>
       </property>
       <property name="text">
        <string>Extension        :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 09pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);
border :0px solid rgb(248, 248, 255);
</string>
       </property>
       <property name="text">
        <string>File Path          :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_textBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>401</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="size_textBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>401</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="path_textBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>401</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="extension_textBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>401</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>601</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border : 1px solid rgb(51, 156, 255);
border-radius : 10px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
background-color: rgb(51, 156, 255);
color : #ffffff
</string>
       </property>
       <property name="text">
        <string>      Progress</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 09pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);
border :0px solid rgb(248, 248, 255);
</string>
       </property>
       <property name="text">
        <string>Operation        :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 09pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);
border :0px solid rgb(248, 248, 255);
</string>
       </property>
       <property name="text">
        <string>Output             :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="output_textBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>401</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="operation_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>401</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248, 255);</string>
       </property>
       <property name="text">
        <string> Show </string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="start_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>430</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 10pt &quot;Calibri&quot;;
	border : 1px solid rgb(51, 156, 255);
	
	background-color: rgb(255, 255, 255);
	border-radius : 10px;
}
QPushButton::hover {
	font: 12pt &quot;Calibri&quot;;
	background-color: rgb(51, 156, 255);
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Start Decryption</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browse_btn">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>430</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 10pt &quot;Calibri&quot;;
	border : 1px solid rgb(51, 156, 255);
	
	background-color: rgb(255, 255, 255);
	border-radius : 10px;
}
QPushButton::hover {
	font: 12pt &quot;Calibri&quot;;
	background-color: rgb(51, 156, 255);
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/photos/Photos/online-security.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Decrypt the Image to get the Message&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
background-color: rgb(255, 255, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>221</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>621</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>86</y>
       <width>461</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
background-color: rgb(248, 248, 255);
border-radius : 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <zorder>groupBox_11</zorder>
    <zorder>groupBox_10</zorder>
    <zorder>groupBox_9</zorder>
    <zorder>groupBox_4</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>groupBox_5</zorder>
    <zorder>groupBox_6</zorder>
    <zorder>label_19</zorder>
    <zorder>label_20</zorder>
    <zorder>groupBox_12</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Decrypt_Window.qrc"/>
 </resources>
 <connections/>
</ui>
